gaudi/README.md

```diff
@@ -0,0 +1,84 @@
+# Setting up HabanaAI Operator
+
+## Overview
+The steps mentioned in this document to deploy and configure [Habana AI Operator](https://catalog.redhat.com/software/container-stacks/detail/64342b3bcbfbb9a6588ce8dd) on Red Hat OpenShift are consolidated and referenced from [HabanaAI Operator for OpenShift](https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/index.html).
+
+## Prerequisities
+- Provisioned RHOCP cluster. Follow steps [here](/README.md).
+
+## Setting up OpenShift Environment
+- Setup Machine Configuration to update Firmware search path using below command. Please note that this step will reboot the nodes.
+```
+oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_firmware_path.yaml
+```
+ 
+- Setup Node Feature Discovery (NFD) instance using below command:
+```
```

Comment:
Can we change it to
## Label Gaudi accelerator nodes With NFD
NFD operator can be used to configure NFD to automatically detect the Gaudi accelerators and label the nodes for the flowing provisioning steps.

Reply:
Made the requested changes.